src/handlers/user.rs

```diff
@@ -18,6 +18,11 @@ use super::swap_beatmaps;
 pub struct Bio {
     pub bio: String,
 }
+impl Bio {
```

Comment:
this is a little overkill for the job. You can directly access the Bio.bio in handler function to check the length.